tools/scylla-nodetool.cc

```diff
@@ -164,7 +164,9 @@ const std::map<std::string_view, std::string_view> option_substitutions{
     {"-et", "--end-token"},
 };
 
-const std::map<operation, operation_func> operations_with_func{
+static auto get_operations_with_func() {
```

Comment:
No need for the `static`, this part of the code is enclosed in an anonymous namespace.

Reply:
ahh, missed that. fixed and repushed.